benchmarks/fixed_pool_stress_test.cpp

```diff
@@ -0,0 +1,90 @@
+//////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+//////////////////////////////////////////////////////////////////////////////
+#include <chrono>
+#include <string>
+#include <random>
+#include <numeric>
+
+#include "umpire/config.hpp"
+#include "umpire/ResourceManager.hpp"
+#include "umpire/Allocator.hpp"
+
+#define CONVERT 1000000 //convert sec (s) to microsec (us)
+
+const int NUM_RND {1000}; //number of rounds (used to average timing)
+const int NUM_ALLOC {512}; //number of allocations used for testing
+const int OBJECTS_PER_BLOCK {1<<11}; //number of blocks of object_bytes size (2048)
+
+void run_test(umpire::Allocator alloc, int SIZE, std::vector<int> &indices, std::string test_name)
```

Comment:
Give the method a more descriptive name, like `test_deallocation_performance`.

Reply:
Done